code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -736,13 +736,13 @@
 //	name = /datum/reagent/mutationtoxin/zombie
 //	id = /datum/reagent/mutationtoxin/zombie
 //	results = list(/datum/reagent/mutationtoxin/zombie = 1)
-//	required_reagents  = list(/datum/reagent/aslimetoxin = 1, /datum/reagent/toxin = 1, /datum/reagent/toxin/bad_food = 1) //Because rotting
+//	required_reagents  = list(/datum/reagent/aslimetoxin = 1, /datum/reagent/drug/krokodil = 10, /datum/reagent/liquidgibs = 10) //Because rotting and liquid gibs are more rotting than bad food
```

Comment:
Why did you change these commented out lines

Reply:
I think I changed those by accident, gonna change it to original state